incuna-sass/_variables.sass

```diff
@@ -0,0 +1,20 @@
+// Set default Incuna-sass variables here
+// This way we can import incuna-sass files above
+// locally defined variables without getting errors
+
+// THESE VARIABLES CAN AND SHOULD BE OVERWRITTEN LOCALLY
+
+// Used in:
+//      forms.sass
+//      em.sass
+//      rem.sass
+$font-base: 16px !default
+
+// Used in:
+//     datepicker.sass
+//     flowplayer.sass
+//     modal.sass
+//     articles.sass
+//     dashboard.sass
+//     reminders.sass
```

Comment:
I don't think we need a list of files the variable is used in. Either we maintain it (painful), or we consider it's not a definitive list, at which point why have it?

I agree that offering information to devs is better than hoping they'll figure it out, but I think finding the usage of a variable is a common enough task that we don't need to pre-empt the search by displaying results here.